### ISSUE TITLE
Unknown type "DatoCmsResolutions"

### ISSUE THREAD

**Issue by zacacollier** [datocms/gatsby-source-datocms#1]
Hello,

Unfortunately, I haven't had any luck getting this gatsby-source to work with Gatsby.

Everything spins up smoothly `gatsby develop` until I hit this error:

```shell
GraphQL Error Unknown type "DatoCmsResolutions". Did you mean "DatoCmsSeoMetaTags", "DatoCmsSiteConnection", "DatoCmsFieldConnection", or "DatoCmsModelConnection"?

  file: /Users/zac/Documents/Dev/datocms-poc-site/.cache/fragments/datocms-seometatags-fragments.js

  1 |
> 2 |   fragment GatsbyDatoCmsResolutions on DatoCmsResolutions {
    |                                        ^
  3 |     base64
  4 |     width
  5 |     height
  6 |     src
  7 |     srcSet
  8 |   }
  9 |
```

It doesn't crash the develop script, but the `data` prop is no longer being passed to any of my components. Granted I'm not entirely not sure if *this* error is what's causing that to happen.

I *can* still browse all the other available types in GraphiQL, but that's about it.

I tried poking around to find a possible culprit - looks like there were some [recent changes](https://github.com/datocms/gatsby-source-datocms/commit/c683180e0c5fe51386407f118d5b029137d3ae84) to `createItemNodes.js`? Perhaps there's an issue there... 🤔

**Comment by stefanoverna** [comment]
Hi @zacacollier, just released 1.0.15 which should solve the problem!

**Comment by zacacollier** [comment]
@stefanoverna excellent, I'll check it out 👏
